RetrospectiveExtension.Frontend/tsconfig.json

```diff
@@ -1,5 +1,6 @@
 ﻿{
   "compilerOptions": {
+    "esModuleInterop": true,
```

Comment:
@msa2984 @focusthenflex Why did you add this config? Is it to fix the error with rendering feedback item?

Reply:
hmm... so from my notes, I see that the rendering issues were fixed when adding the "strictNullChecks" option -also, I think this shouldn't matter anyways since we're using a separate tsconfig for testing anyways... so this change should be reverted

Reply:
We actually use it for creating the bundle file in production as well. If you try removing this flag on our  current`dev` branch and test the production bundle (`npm run build:p`), there will be an issue when creating feedback item. We can talk more about it in our stand up but it's good for now.